api/getTriviaQuestion.js

```diff
@@ -0,0 +1,37 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+const axios = require("axios");
+const hmacSHA512 = require("crypto-js/hmac-sha512");
+const {REACT_APP_GAME_DB_USER, REACT_APP_GAME_DB_HOST, REACT_APP_GAME_DB_NAME,REACT_APP_GAME_DB_PASSWORD, REACT_APP_GAME_DB_PORT} = process.env;
+
+const Pool = require("pg").Pool;
```

Comment:
Please see my other comment on using a connection string here. Also this is something we should initialize in a different file and pass the instance to any file that needs it instead of creating lots of connections to the same db.